java/src/jmri/implementation/AbstractNamedBean.java

```diff
@@ -2,6 +2,7 @@
 
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import javax.annotation.CheckReturnValue;
+import javax.annotation.OverridingMethodsMustInvokeSuper;
```

Comment:
For a lot of the methods marked with this annotation, might it not be preferable to make those methods final?